graphics/cursorman.h

```diff
@@ -68,22 +68,25 @@ class CursorManager : public Common::Singleton<CursorManager> {
 	 * can be safely freed afterwards.
 	 *
 	 * @param buf		New cursor data.
+	 * @param mask		New cursor data.
```

Comment:
"New cursor mask data"

Reply:
That param desc actually appears twice, and this one is in the wrong place from a prior version.  Removed it.